src/components/tracksList.vue

```diff
@@ -1,39 +1,46 @@
-<script setup lang="ts">
-import { ref, Ref } from "vue";
-import { TrackApi, TrackModel } from "@bcc-code/bmm-sdk-fetch";
+<script lang="ts" setup>
+import filters from "@/utils/filters";
+import { useAuth0 } from "@auth0/auth0-vue";
+import { onMounted, ref } from "vue";
 
-import ProtectedImage from "./ProtectedImage.vue";
+// import ProtectedImage from "./ProtectedImage.vue";
 
-const tracks: Ref<TrackModel[]> = ref([]);
+const props = defineProps<{
```

Comment:
Please be more specific than just `any`. By just making the type `any`, this can be anything. A boolean, array, function, null, etc...